metabase/metabase
# Show query error in the parameter modal

Epic https://github.com/metabase/metabase/issues/26898

How to verify:
- Create new dashboard with a parameter
- Edit -> Select parameter -> Dropdown list -> Edit
- Select a saved question and pick a column. You'll see list of values
- Go to the network tab and block the url for fetching parameter values (should be the last call)
- Pick a different column in the question. You should see this new error

<img width="1408" alt="Screenshot 2023-02-07 at 17 27 10" src="https://user-images.githubusercontent.com/8542534/217291350-049e99d5-6f3c-451c-92b7-9e204acabbff.png">


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/28119)
